assembl/models/action.py

```diff
@@ -305,8 +310,8 @@ def send_post_to_socket_ts(mapper, connection, target):
         & (_dpt.c.post_id == Content.__table__.c.id)
         & (_actt2.c.type ==
            DisagreeSentimentOfPost.__mapper_args__['polymorphic_identity'])
-        & (_actt2.c.tombstone_date == None)
-        ).correlate_except(_actt2, _dpt), deferred=True)
+        & (_actt2.c.tombstone_date is None)
```

Comment:
revert this change
really don't modify this file, flake8 branch already fixes the errors.